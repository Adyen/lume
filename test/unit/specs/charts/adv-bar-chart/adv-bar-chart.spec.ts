import { shallowMount} from '@vue/test-utils';
import { data } from '../../mock-data';
import BarChart from '@/charts/adv-bar-chart/adv-bar-chart.vue';

describe.skip('bar.vue', () => {
    test('mounts component and sets prop values', () => {
        const wrapper = shallowMount(BarChart, {
            propsData: { data }
        });

        expect(wrapper.find('[data-j-adv-bar-chart]')).toBeTruthy();
    });
});
