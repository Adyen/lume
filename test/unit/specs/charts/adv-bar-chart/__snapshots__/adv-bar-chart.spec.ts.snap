// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`adv-bar-chart.vue mounts component and sets prop values 1`] = `<!---->`;
